run-tests.sh

```diff
@@ -331,6 +355,22 @@ for arch in $ARCHS; do
                 fi
                 i=$(($i+1))
             done
+            for test in $TESTS_CFGUARD; do
+                file=$test.exe
+                OUT=cmdoutput
+                rm -f $OUT
+                if $RUN $test.exe check_enabled; then
+                    $RUN $test.exe normal_icall
+                    $RUN $test.exe invalid_icall_nocf || [ $? = 2 ]
+                    # We want to check the exit code to be 0xc0000409
+                    # (STATUS_STACK_BUFFER_OVERRUN aka fail fast exception).
+                    # MSYS2 bash does not give us the full 32-bit exit code, so
+                    # we have to rely on cmd.exe to perform the check.
+                    # (This probably doesn't work on WINE, but WINE doesn't
+                    # support CFG anyway, at least not for now...)
+                    $RUN cmd //c "$test.exe invalid_icall & if errorlevel -1073740791 (if not errorlevel -1073740790 (exit 0)) & exit 1"
```

Comment:
What's the deal with the second check `if not errorlevel -1073740790` here? The former check, `-1073740791`, corresponds to the expected `0xc0000409`, but what's the second one?

Reply:
`if errorlevel N` is effectively `if errorlevel >= N`, so I need to do 'errorlevel >= N and not errorlevel >= N+1' to check for equality. (I can't check equality of `%errorlevel%` directly because of how cmd's variable substitution works.)

Reply:
Ah, I see. That's rather nonobvious and clearly worthy of a comment!

Reply:
Oh, maybe I actually can use a more direct equality comparison if I enable delayed expansion, let me try that.